pages/base.py

```diff
@@ -49,24 +53,28 @@ def is_user_loggedin(self):
     def click_browserid_login(self):
         self.selenium.find_element(*self._browserid_login_locator).click()
 
-    def login(self, email, password):
+    def login(self, email):
         self.click_browserid_login()
-        from browserid import BrowserID
-        pop_up = BrowserID(self.selenium, self.timeout)
-        pop_up.sign_in(email, password)
+        from pages.auth_zero import AuthZero
+        auth_zero = AuthZero(self.base_url, self.selenium)
+        auth_zero.request_login_link(email)
+        login_link = restmail.get_login_link(email)
+        self.open(login_link)
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.is_user_loggedin)
 
-    def create_new_user(self, email, password):
+    def create_new_user(self, email):
         self.click_browserid_login()
-        from browserid import BrowserID
-        pop_up = BrowserID(self.selenium, self.timeout)
-        pop_up.sign_in(email, password)
-
+        from pages.auth_zero import AuthZero
+        auth_zero = AuthZero(self.base_url, self.selenium)
+        auth_zero.request_login_link(email)
+        login_link = restmail.get_login_link(email)
+        self.open(login_link)
```

Comment:
This is duplicated, it would be good to remove that duplication. This method could just call `self.login` but return the `Register` page object.